lib/carrierwave/sequel.rb

```diff
@@ -11,6 +11,43 @@ def mount_uploader(column, uploader, options={}, &block)
       raise "You need to use Sequel 3.0 or higher. Please upgrade." unless ::Sequel::Model.respond_to?(:plugin)
       super
 
+      mod = Module.new
+      include mod
+      mod.class_eval <<-RUBY, __FILE__, __LINE__+1
+        def #{column}=(new_file)
+          if !(new_file.blank? && send(:#{column}).blank?)
+            modified!
+          end
+          super
+        end
+
+        def remove_#{column}!
```

Comment:
Not sure. My goal was just to make removing work properly (and in combination with sequel #update), which now it does. Unfortunately I don't have enough time at the moment to do more. Ideally this repo should, in my opinion, follow the ActiveRecord implementation as close as possible. Questions like the above should be raised in the carrierwave repo where the AR integration is. Then we can bring the same change into the Sequel integration.